### ISSUE TITLE
Video-AMME CI accuracy differs between H20 and H200

### ISSUE THREAD

**Issue by PasserBy4** [sgl-project/sglang-omni#458]
`tests/test_model/test_qwen3_omni_videoamme_ci.py` reports different accuracy on the two GPUs with the same model, dataset, and config:

| GPU  | Correct | Accuracy   | CI result (threshold = 0.6667) |
|------|---------|------------|-----|
| H20  | 20/30   | **66.67%** | pass |
| H200 | 18/30   | **60.00%** | **fail** |

H20 passes consistently and H200 fails consistently. Needs investigation: is this random run-to-run jitter, or a real accuracy regression on H200?

## Findings

1. **Only 2 samples flip** (003-2, 008-2), both wrong on H200 and correct on H20. `2/30 = 6.67%` — exactly the gap.

2. **Inputs are identical.** `prompt_tokens` matches per-sample for all 30 cases. Server logs show both machines decode each video via `torchcodec` with identical `total_frames` and `video_fps`. → preprocessing is not the cause.

3. **MoE Triton kernel loads device-specific configs**, and both machines fall back to different older Triton versions:
   ```
   H20:  triton_3_3_1/E=128,N=768,device_name=NVIDIA_H20.json
   H200: triton_3_2_0/E=128,N=768,device_name=NVIDIA_H200.json
   ```
   Different tile/block sizes produce different bf16 reduction orders. Across 48 MoE layers this is enough to flip tie-break tokens at the logits layer.

   Both also emit `Using MoE kernel config with down_moe=False. Performance might be sub-optimal!` — the down-projection MoE has no matching config and uses a default.
